### PR TITLE
solution test deployment

### PR DESCRIPTION
**Pull request by willwildred**
Hi! Before you check my solution: some tests doesn't pass, but my solution 1:1 as it should be. Menthor said to ignore them, so please, don't look at them while reviewing, thanks!

- [DEMO LINK](https://willwildred.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://willwildred.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,14 +1,75 @@
...
+          <li>
+            <a href="#" class="nav-links is-active">
+              Apple
+            </a>
+          </li>
+          <li>
+            <a href="#" class="nav-links">
+              Samsung
+            </a>
+          </li>
```
Comment (anastasiia-tilikina) [inline src/index.html:35]:
```suggestion
          <li>
            <a href="#" class="nav-links is-active">
              Apple
            </a>
          </li>
          
          <li>
            <a href="#" class="nav-links">
              Samsung
            </a>
          </li>
```
It is a good practice to add blank line between multiline siblings.
Pls fix everywhere

File: src/index.html [hunk 2]
```diff
@@ -1,14 +1,75 @@
...
+            <a href="#" class="nav-links">
+              Smartphones
+            </a>
+          </li>
+          <li>
+            <a
+            href="#"
+            class="nav-links"
+            data-qa="hover"
+            >
```
Comment (anastasiia-tilikina) [inline src/index.html:46]:
```suggestion
            <a
              href="#"
              class="nav-links"
              data-qa="hover"
            >
```
please pay attention to your code formatting

Comment (anastasiia-tilikina) [reply src/index.html:46]: check everywhere

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,61 @@
...
+  list-style-type: none;
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.nav-links {
+  display: flex;
+  align-items: center;
+  height: 60px;
```
Comment (anastasiia-tilikina) [inline src/style.css:33]: it's not a good idea to hardcode height. use proper padding instead

**Review by anastasiia-tilikina — CHANGES_REQUESTED** [review]
Good job!

let's just fix a couple of issues here

File: src/index.html [hunk 4]
```diff
@@ -1,14 +1,82 @@
...
+  <!-- #region HEAD -->
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <!-- #region Fonts -->
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <!-- #endregion -->
```
Comment (OleziO) [inline src/index.html:12]: Remove all comments from the code.

File: src/index.html [hunk 5]
```diff
@@ -1,14 +1,82 @@
...
+    <header class="header">
+      <a href="#" class="logo">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+
+      <!-- #region NAV LINKS -->
+      <nav class="nav">
+        <ul class="nav-menu">
+          <li>
+            <a href="#" class="nav-links is-active">
```
Comment (OleziO) [inline src/index.html:27]: Make links more meaningful. Example: #apple, #samsung, etc.

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,62 @@
+html {
+  --color: #00ACDC;
+  --header-padding: 23px 0;
```
Comment (OleziO) [inline src/style.css:3]: Move all variables to the :root selector.

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,62 @@
...
+}
+
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```
Comment (OleziO) [inline src/style.css:9]: Move to the * selector and remove it from all other places.

**Review by OleziO — CHANGES_REQUESTED** [review]
image
Fix all tests.

**Review by l4st1m0za — APPROVED** [review]
Great job!

**Pull request closed without merging**